tests/test_DateTime.js

```diff
@@ -43,7 +43,7 @@ tests.test_DateTime = function (t) {
     localTZ = Math.abs(localTZ);
     localTZ = direction + ((localTZ < 10) ? "0" : "") + localTZ;
     testTimestamp = isoTimestamp("2005-2-3T22:01:03" + localTZ);
-    var testDateTimestamp = new Date(2005,1,3,22,1,3);
+    var testDateTimestamp = new Date(2005,1,3,22,31,3);
```

Comment:
I don't think this is correct, it fails here. If the existing test fails for you, what time zone is your computer set to?

```suggestion
    var testDateTimestamp = new Date(2005,1,3,22,1,3);
```

Reply:
@etrepum  I am in IST timezone.

Reply:
Great catch! This looks like a longstanding issue with the test suite that has been there for over a decade. The implementation does not seem to have any issue with IST.